### PR TITLE
Updated formatting on "Add a Submit Button to a Form" article.

### PR DESCRIPTION
**Pull request by nsuchy**
This PR updates the formatting on the "Add a Submit Button to a Form" article.

### CODE REVIEW

File: src/pages/certificates/add-a-submit-button-to-a-form/index.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
 ---
 title: Add a Submit Button to a Form
 ---
-You will need to create a `button` element. Here's an example submit button:
+You will need to create a `button` element.
 
-    <button type="submit">this button submits the form</button>
+**Here's an example submit button:**
+```
```
Comment (texas2010) [inline src/pages/certificates/add-a-submit-button-to-a-form/index.md]: you forget to add html in the backtick. ha

**Comment by nsuchy** [comment]
In this case the lack of HTML beside the backticks was intentional as it was demoing the button using the code container as part of the design. I can change it if you'd like but it looked nice when in development mode.

**Comment by texas2010** [comment]
oh okay.

**Comment by pawan-mehta** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  ossbackup2@gmail.com
    Domain oss-india.net has exceeded the max emails per hour (101/100 (101%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from mail-oi0-f49.google.com ([209.85.218.49]:55152)
	by server14.hostingraja.in with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <pawan.ossindia+caf_=pawan=oss-india.net@gmail.com>)
	id 1e5JRr-001XQO-0z
	for pawan@oss-india.net; Fri, 20 Oct 2017 04:08:08 +0530
Received: by mail-oi0-f49.google.com with SMTP id a132so17412426oih.11
        for <pawan@oss-india.net>; Thu, 19 Oct 2017 15:37:34 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:delivered-to:dkim-signature:dkim-signature:date
         :from:reply-to:to:cc:message-id:in-reply-to:references:subject
         :mime-version:content-transfer-encoding:precedence:list-id
         :list-archive:list-post:list-unsubscribe;
        bh=efwrxVydEbA58MJkG9XG9rmxhQc3E/wj/ZIo9pZfyp8=;
        b=pqNU4PgPceokLUoY7nc9jRBrqXtMyOR2AjTxNEvwrKRmnwa71F4oVMJ1H2POW6BcZY
         L1K4xmC/PXF/73ZA3Fwsxgmpc6ah5cL10Q1ZbaeXQG7IVY4rF7EiIJFw2HsTwpZSWzio
         Tim+pa00xYUKPW01B6qUBLRUlNQiO+rr60mvaFBvs0O8pNHbqrSUyh4H5JsPQGy4ffYl
         plsQIaPY3IXMllK/e7Enh1FbuytLOiI6nAvaB8bos+NIFKRy+m5FhXuTO/20Q71G2VC9
         yaieZSOLJKrkPdLjtSll7lpqW75t+5dlKmpl8LyTqiyUc2Xjtv4HdY0qwQ6icjGK6h18
         iAOw==
X-Gm-Message-State: AMCzsaXaCkTHIZN2rpjW923Z/ygTT5sPPJzfH2FrbaMsDMbKGSmR/jJ8
	LWCOIcFR9l0UfB89U/tBYRpYNFWxX0A8po78XRAjqAJPL1lbfW0=
X-Received: by 10.157.37.177 with SMTP id q46mr2063053ota.55.1508452632242;
        Thu, 19 Oct 2017 15:37:12 -0700 (PDT)
X-Forwarded-To: pawan@oss-india.net
X-Forwarded-For: pawan.ossindia@gmail.com pawan@oss-india.net
Delivered-To: pawan.ossindia@gmail.com
Received: by 10.168.68.11 with SMTP id r11csp4391973ofa;
        Thu, 19 Oct 2017 15:37:11 -0700 (PDT)
X-Google-Smtp-Source: ABhQp+RCMQDI7dz0LtiO6V4Zur67r8Wkxh+reyAF9f4YS3ixc42vc/fFgSb9okdkLHPN4VyPPQVa
X-Received: by 10.55.217.203 with SMTP id q72mr4526874qkl.245.1508452631616;
        Thu, 19 Oct 2017 15:37:11 -0700 (PDT)
ARC-Seal: i=1; a=rsa-sha256; t=1508452631; cv=none;
        d=google.com; s=arc-20160816;
        b=AINJ0k6F7uKcpXC36WjDeTahISQULCm7CK4QAArzces0d4iTcR0j5P8O0vvoUPmWlq
         eAkxhEWP03zZgtZGdQb9GuHbmQIBRFDDRAca6lWf3THThzjp0Pa7kf9CGhsVVX7KftuR
         RAUPgo5lEZdb4X9lR76qOm4OWJ1EWzmeDSJw2xQ0REXAVMZGxUQWv4+5m7aEGxa5rbuc
         nxWraWyLMghXEMK1HucKBliyFUTl1pVO9Az0o1CQ7gZ7M7/rS7hOgE0vPazHMgmPcM8w
         MTV6GYT3TkYKKs2hBRn9I6KFRVJEwkiaH50y0XY7SpnH3bMBmgd0PB6iaJasfswEEb2h
         ju8Q==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=list-unsubscribe:list-post:list-archive:list-id:precedence
         :content-transfer-encoding:mime-version:subject:references
         :in-reply-to:message-id:cc:to:reply-to:from:date:dkim-signature
         :dkim-signature:arc-authentication-results;
        bh=efwrxVydEbA58MJkG9XG9rmxhQc3E/wj/ZIo9pZfyp8=;
        b=QPvpY4pobjeSZArSHL3JFy2vOgWECijtinlChBCWJcwHPQ6FLYjE285YtSLnidJANG
         dwAUXP8bcaRV89ReBkxCMbm7C/sVp0W2KuoOTBGUXbphwVvnr+yO7h93LBBPlOnghObO
         WaaTvRbYZMSVggbx3kIgvRwZXhePBVdgKCiDMXACwXIOYOx/N8Xourmrh7LlRAyZZSZD
         tAn58YbDQW2V8xbcrrT+KjEA/uoZpxXIsq1DrRg5h5ANBxpApLcTSE8MAD3E3Lp3tA1m
         htc/Dea4MDio/Z6bPSRJGmoGK8f52akJYuBOSOr4zoD3xmy3uQgdv/9M2I9lrb9Lj6Zv
         LRLA==
ARC-Authentication-Results: i=1; mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=qeurSsfW;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=UNxtSI5P;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.10 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
Received: from o5.sgmail.github.com (o5.sgmail.github.com. [192.254.113.10])
        by mx.google.com with ESMTPS id r1si14069277qtf.38.2017.10.19.15.37.11
        for <pawan.ossindia@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Thu, 19 Oct 2017 15:37:11 -0700 (PDT)
Received-SPF: pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.10 as permitted sender) client-ip=192.254.113.10;
Authentication-Results: mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=qeurSsfW;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=UNxtSI5P;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 192.254.113.10 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=GBPe2xoobkGegXhAIOAVa9BTZD0=; b=qeurSsfWXsgCwHxA
	MlEuZsOmY/lhpbSF7o1nHJAVw6h93yyNX+gwIpPA9bhkOV12rxxOuJ817f3BJWBF
	p/u+J9/plTieelFHIdDOBlHSGzu/m9BGMiQsY+fbxO1ZwmcjNEqcudJDEMQn3Hjd
	TaBjStLnV+5iQf7txHuS7+fTL0s=
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=sendgrid.info; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe:x-feedback-id; 
	s=smtpapi; bh=GBPe2xoobkGegXhAIOAVa9BTZD0=; b=UNxtSI5PqxO++hHHKG
	oga5CfdzoTtj1Mss2SVOH4GUVUqIn54fCepEqscE/jU9Gf/Sq9JWpIEuRe8shV0F
	PVapCwnIDPlVMwUPtVcProgU/FbJ1c9epG61kaIrizlP9o0PjQtYofQKahekv5NZ
	iWkS6AOUWq0hxEFm3DaaBzDYQ=
Received: by filter0144p1iad2.sendgrid.net with SMTP id filter0144p1iad2-13569-59E92917-16
        2017-10-19 22:37:11.252465548 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0002p1iad2.sendgrid.net (SG) with ESMTP id 4y3pqcIxT2-2bvGeeNz8xw
	for <pawan.ossindia@gmail.com>; Thu, 19 Oct 2017 22:37:11.213 +0000 (UTC)
Date: Thu, 19 Oct 2017 22:37:11 +0000 (UTC)

**Pull request merged**